test/plugins/test-trace-hapi.js

```diff
@@ -308,6 +308,41 @@ describe('hapi', function() {
           });
         });
       });
+
+      it('should end spans when client aborts request', function(done) {
```

Comment:
Would it be possible to include comparable tests for the other frameworks even if no logical changes are required in the hooks? How does an aborted request surface for those frameworks?